app/javascript/pages/country/widgets/widget-tree-cover-gain/widget-tree-cover-gain-reducers.js

```diff
@@ -1,23 +1,39 @@
 export const initialState = {
   isLoading: true,
-  totalAmount: 0,
-  percentage: 0,
-  locations: [
+  gain: 0,
+  treeExtent: 0,
+  indicators: [
```

Comment:
I know you are making a helper now, but a note to make sure we handle the selector list from constants and a single source of truth and not in each widget.